tests/scripts/run-xunit-perf.py

```diff
@@ -71,7 +71,7 @@ def validate_arg(arg, check):
             raise ValueError('Argument: %s is not valid.' % (arg))
 
     coreclrPerf = os.path.join(os.getcwd(), args.coreclrPerf)
-    validate_arg(coreclrPerf, lambda item: os.path.isdir(item))
+    validate_arg(coreclrPerf, lambda item: os.path.isdir(item) or os.path.isfile(item))
```

Comment:
This is rather error prone. It would be nicer if you add a parameter for the assembly file name case and handle it separately.

Reply:
No problem